### ISSUE TITLE
Stuck on Successfully opened dynamic library cudnn64_7.dll

### ISSUE THREAD

**Issue by jtpaquet** [CorentinJ/Real-Time-Voice-Cloning#654]
I did all the installations and when I record myself and press on Synthetize only I get stuck on this line in the CLI:

`2021-02-10 00:45:37.068088: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll`

The GUI stop responding and it says:
`Loading the synthesizer synthesizer\saved models\logs-pretrained\taco_pretrained\tacotron_model.ckpt-278000`

One thing I could add is that I can't load the package `webrtcvad`:

`E:\Other\Voices\Real-Time-Voice-Cloning-master\encoder\audio.py:13: UserWarning: Unable to import 'webrtcvad'. This package enables noise removal and is recommended.
  warn("Unable to import 'webrtcvad'. This package enables noise removal and is recommended.")
2021-02-10 00:43:29.707998: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
Arguments:
    datasets_root:    None
    enc_models_dir:   encoder\saved_models
    syn_models_dir:   synthesizer\saved_models
    voc_models_dir:   vocoder\saved_models
    low_mem:          False
    seed:             None
    no_mp3_support:   False

Warning: you did not pass a root directory for datasets as argument.
The recognized datasets are:
        LibriSpeech/dev-clean
        LibriSpeech/dev-other
        LibriSpeech/test-clean
        LibriSpeech/test-other
        LibriSpeech/train-clean-100
        LibriSpeech/train-clean-360
        LibriSpeech/train-other-500
        LibriTTS/dev-clean
        LibriTTS/dev-other
        LibriTTS/test-clean
        LibriTTS/test-other
        LibriTTS/train-clean-100
        LibriTTS/train-clean-360
        LibriTTS/train-other-500
        LJSpeech-1.1
        VoxCeleb1/wav
        VoxCeleb1/test_wav
        VoxCeleb2/dev/aac
        VoxCeleb2/test/aac
        VCTK-Corpus/wav48
Feel free to add your own. You can still use the toolbox by recording samples yourself.`

The CLI since the loading checkpoint:

`Loading checkpoint: synthesizer\saved_models\logs-pretrained\taco_pretrained\tacotron_model.ckpt-278000
2021-02-10 00:44:03.798882: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2021-02-10 00:44:03.806050: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2021-02-10 00:44:03.831446: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties:
name: GeForce RTX 3080 major: 8 minor: 6 memoryClockRate(GHz): 1.785
pciBusID: 0000:21:00.0
2021-02-10 00:44:03.831538: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2021-02-10 00:44:03.835372: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2021-02-10 00:44:03.838306: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2021-02-10 00:44:03.839463: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2021-02-10 00:44:03.843225: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2021-02-10 00:44:03.845400: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2021-02-10 00:44:03.852431: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2021-02-10 00:44:03.852557: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2021-02-10 00:44:58.939104: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-02-10 00:44:58.939228: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0
2021-02-10 00:44:58.941133: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N
2021-02-10 00:44:58.941796: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 8551 MB memory) -> physical GPU (device: 0, name: GeForce RTX 3080, pci bus id: 0000:21:00.0, compute capability: 8.6)
2021-02-10 00:44:59.515354: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2021-02-10 00:45:37.068088: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll`

Am I missing something?

thx

**Comment by blue-fish** [comment]
Tensorflow 1.x is not compatible with NVIDIA 30xx GPUs.
I have a pull request to remove this dependency, it is in the final stages of review.
It's already released in my fork if you don't want to wait.
https://github.com/blue-fish/Real-Time-Voice-Cloning

**Comment by jtpaquet** [comment]
That was it, I tried on my roommate's pc with a GTX 1060 and it worked with no problems, thanks

**Comment by blue-fish** [comment]
Closing this issue, #472 is merged and removes the dependency on tensorflow.
